Brewfile

```diff
@@ -9,10 +9,25 @@ tap 'homebrew/services'
 
 brew 'hub'
 
+brew 'awscli'
+brew 'gpg'
+brew 'opensc'
```

Comment:
I don't believe this is needed anymore since macOS has native PIV support these days.